packages/@react-stately/virtualizer/src/Virtualizer.ts

```diff
@@ -865,7 +865,7 @@ export class Virtualizer<T extends object, V, W> {
     // Apply layout infos to visible views
     for (let view of this._visibleViews.values()) {
       let cur = view.layoutInfo;
-      if (cur) {
+      if (cur.key) {
```

Comment:
I'm looking back at the error and it says can't get `key` from a `null` `cur`. This fix is saying we also have an issue where `cur` exists, but has a `null` `key`, is that correct?
How does a `layoutInfo` exist, but doesn't have a `key`?

Is this more of a follow up question related to fixing the underlying issue(s)?

Reply:
Fixed this to be `!= null` like Rob suggested. And yeah, this isn't a perfect solution, but we wanted to prevent crashing for now, and we can do a follow-up after a lot of the TableView refactor that's going on now gets merged.